lib/modules/platform/azure/index.ts

```diff
@@ -144,23 +146,34 @@ export async function getRawFile(
       return null;
     }
 
+    let item: GitItem | undefined;
     const versionDescriptor: GitVersionDescriptor = {
       version: branchOrTag,
     } satisfies GitVersionDescriptor;
-
-    const item = await azureApiGit.getItem(
-      repoId, // repositoryId
-      fileName, // path
-      undefined, // project
-      undefined, // scopePath
-      undefined, // recursionLevel
-      undefined, // includeContentMetadata
-      undefined, // latestProcessedChange
-      undefined, // download
-      branchOrTag ? versionDescriptor : undefined, // versionDescriptor
-      true, // includeContent
-    );
-
+    // Try to get file from repo with branch, if not found, then try with tag #36835
+    for (const versionType of [GitVersionType.Branch, GitVersionType.Tag]) {
```

Comment:
```suggestion
    for (const versionType of [GitVersionType.Tag, GitVersionType.Branch]) {
```

Let's reverse the order to try the tag first, as that was what was intended in the docs and how it works in other platforms

Reply:
I reversed the order in this commit, and also updated the tests and comments:
https://github.com/renovatebot/renovate/pull/36874/commits/7adb1a908dbdd0d29d800a9135758fe3d98e58ea
Let me know if this is sufficient.